frontend/components/common/table.jsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const MyTable = ({ header, body, width }) => (
+  <Table width={width}>
+    <HeaderSlot>{header()}</HeaderSlot>
+    <BodySlot>{body()}</BodySlot>
```

Comment:
header, body가 함수라면 getTableHead, getTableBody 같은 이름도 좋아보입니다
```suggestion
    <HeaderSlot>{getTableHead()}</HeaderSlot>
    <BodySlot>{getTableBody()}</BodySlot>
```